package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "ion-study-template",
+  "version": "0.0.1",
+  "scripts": {
+    "build": "rollup -c",
+    "dev": "rollup -c -w",
+    "lint": "eslint . && web-ext lint",
+    "test": "npm run test-addon"
```

Comment:
nit: this will probably 💥  since there is no `test-addon` script listed here to run.

Reply:
Whoops, yup :) I'll be adding an example test in later commits. I'm trying to figure out the most recent versions of RollupJS break the `npm run build` :-(

Reply:
I removed the test command. It will come as a separate PR.